crates/ruff_linter/src/rules/flake8_pyi/rules/unrecognized_platform.rs

```diff
@@ -39,7 +40,7 @@ use crate::registry::Rule;
 /// ```
 ///
 /// ## References
-/// - [PEP 484](https://peps.python.org/pep-0484/#version-and-platform-checking)
+/// - [Static typing with Python: Type Stubs: Version and Platform Checks](https://typing.readthedocs.io/en/latest/source/stubs.html#version-and-platform-checks)
```

Comment:
nit: maybe just "Python Typing Documentation: Version and Platform Checks" ? Or "Typing Documentation: Version and Platform Checks"